src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-anchor.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import AlloyEditor from 'alloyeditor';
+
+export default class EzBtnAnchor extends Component {
+    constructor(props) {
+        super(props);
+
+        this.getStateClasses = AlloyEditor.ButtonStateClasses.getStateClasses;
+    }
+
+    static get key() {
+        return 'ezanchor';
+    }
+
+    /**
+     * Lifecycle. Renders the UI of the button.
+     *
+     * @method render
+     * @return {Object} The content which should be rendered.
+     */
+    render() {
+        if (this.props.renderExclusive) {
+            return <AlloyEditor.EzBtnAnchorEdit {...this.props} />;
+        }
+
+        const cssClass = 'ae-button ez-btn-ae--anchor ez-btn-ae ' + this.getStateClasses();
+        const label = Translator.trans(/*@Desc("Anchor")*/ 'anchor_btn.label', {}, 'alloy_editor');
```

Comment:
Not sure, are we missing generated translation here and in `ez-btn-anchoredit.js`?